martin/src/config/file/cache.rs

```diff
@@ -18,8 +21,11 @@ impl CacheConfig {
     #[must_use]
     pub fn create_tile_cache(&self) -> Option<martin_core::tiles::TileCache> {
         if self.tile_cache_size_mb > 0 {
+            log::info!(
+                "Initializing tile cache with maximum size {} MiB",
```

Comment:
can we please just use `MB`? Makes life soo much more readable :)

Reply:
Read:
- change to base `1000` too or
- stick with base `1024` and incorrectly/windows-style mark this as `MB`

Reply:
Our frontend also uses `MiB`, so better change there too